### ISSUE TITLE
Unable to reach Buttons via Keyboard Navigation in certain Dialogs (Devices,Weather) 

### ISSUE THREAD

**Issue by condoratberlin** [XCSoar/XCSoar#394]
Affected Versions:
 - Master / Stable

Affected Dialogs
- Weather (Remove)
- Devices (Edit, Disable, Reconnect, Monitor, Flight Download, Debug)
- Turn Points (Make Finish, Move TP Up, Move TP Down)

### Description

1. Issue
In the ListWidget dialog, there is normally a default action when selecting an item. 
An example in the weather (Metar and TAF) dialog is that you navigate to the entry and you hit "enter" and it will display that entry. 

This does not happen in the "Devices" dialog. There is no default action when hitting enter. 

2. Issue
-  if there is for example 3 entries: 
 its not possible to select the middle entry and press the "Remove" Button

### Expected Behavior
Moving to the entry in the list and then moving either left or right should jump to the buttons. There should also be a default action when hitting return.
